README.md

```diff
@@ -1,21 +1,45 @@
 # BPMN Visu JS
-A JavaScript library to visualize process execution data. Diagrams are displayed from [BPMN](https://www.omg.org/spec/BPMN/2.0.2/PDF) files. With additional display
-options for execution data (highlight some transitions, counters, and more). With interactive capacities (mouse hover,
-click).
+A JavaScript library to visualize [BPMN](https://www.omg.org/spec/BPMN/2.0.2/PDF) diagrams as well as meta-information about process execution, which the user can interact with.
+  - BPMN diagrams are read from .bpmn files that may not only come from Bonita but rather be created from any BPM tool or even created manually. This means that it will be able to display more BPMN 2.0 elements than those supported by Bonita.
```

Comment:
I don't think we need to talk about Bonita here, because the repository will be in the organisation Process Analytics (no Bonitasoft here). It could be confusing for the user, who don't know we we talk about it.